### PR TITLE
Composer project setup

### PR DESCRIPTION
**Pull request by hexus**
PSR-4 autoloading from the `src` directory. Namespaced classes beginning with the namespace `HeraldryEngine` will be autoloaded from here.

PSR-2 code style in some places.

Included jQuery and path-data-polyfill distributives in scripts/vendor.

### CODE REVIEW

**Comment by hexus** [comment]
This isn't finished yet, just opening the PR as a WIP.

**Comment by hexus** [comment]
This should be good to go. :+1: Not all pages have been fully converted to use the new bootstrap approach, but feel free to continue on with that.

File: .gitignore [hunk 1]
```diff
@@ -55,3 +55,6 @@ scripts/jquery-3.2.1.min.js
 
 # Composer dependencies
 /vendor/
+
+#protect Chris from himself
```
Comment (hexus) [inline .gitignore]: I wonder how many places my name will be referenced in the project before version 1.0.0 😂

**Comment by qxzkjp** [comment]
Changes merged into working branch.

**Pull request closed without merging**